src/test/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtilTest.java

```diff
@@ -324,6 +324,22 @@ public void test_getShowDeletedValueFalse() {
         assertThat(response.equals("false")).isTrue();
     }
 
+    @Test
+    public void test_getReturnRolesValueTrue() {
+        String returnRoles = "True";
+        String response = RefDataUtil.getReturnRolesValue(returnRoles);
+
+        assertThat(response.equals("true")).isTrue();
+    }
+
+    @Test
+    public void test_getReturnRolesValueFalse() {
+        String returnRoles = "false";
+        String response = RefDataUtil.getReturnRolesValue(returnRoles);
+
+        assertThat(response.equals("false")).isTrue();
+    }
+
```

Comment:
@Test
     public void test_getReturnRolesValueTrue() {
          assertThat(RefDataUtil.getReturnRolesValue("True")).isTrue();
          assertThat(RefDataUtil.getReturnRolesValue("true")).isTrue();
          assertThat(RefDataUtil.getReturnRolesValue("TRUE")).isTrue();
          assertThat(RefDataUtil.getReturnRolesValue("False")).isFalse();
          assertThat(RefDataUtil.getReturnRolesValue("false")).isFalse();
          assertThat(RefDataUtil.getReturnRolesValue("FALSE")).isFalse();
          assertThat(RefDataUtil.getReturnRolesValue("invalid")).isTrue();

     }

Reply:
can be written in 1 method only

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/603/commits/2413e2abe279f9a8b36b1f3f1f56d767fd9f004d